backend/SlothOrganizer/SlothOrganizer.Presentation/Controllers/DashboardController.cs

```diff
@@ -18,15 +19,31 @@ public DashboardController(IDashboardService dashboardService)
         }
 
         [HttpPost]
-        public async Task<DashboardDto> Create([FromBody] NewDashboardDto newDashboard)
+        public async Task<ActionResult<DashboardDto>> Create([FromBody] NewDashboardDto newDashboard)
         {
-            return await _dashboardService.Create(newDashboard);
+            var email = GetCurrentUserEmail();
+            if (email is null)
+            {
+                return Unauthorized();
+            }
+            return Ok(await _dashboardService.Create(newDashboard, email));
         }
 
-        [HttpGet("{userId}")]
-        public async Task<List<DashboardDto>> Get(long userId)
+        [HttpGet]
+        public async Task<ActionResult<List<DashboardDto>>> Get()
         {
-            return await _dashboardService.Get(userId);
+            var email = GetCurrentUserEmail();
+            if (email is null)
+            {
+                return Unauthorized();
+            }
+            return Ok(await _dashboardService.Get(email));
+        }
+
+        private string? GetCurrentUserEmail()
+        {
+            var identity = HttpContext.User.Identity as ClaimsIdentity;
```

Comment:
Why do you use email? It may cause issues in future. Id is what you will defenitely have always have in your application.

Reply:
Added id to claims and used it for dashboard